backend_developer.md

```diff
@@ -17,11 +17,11 @@ We are looking for a backend developer to build new features and maintain the ex
 
 Our stack buzzwords are Elixir, Ubuntu, PostgreSQL, Vagrant, json:api, Zigbee and Wireless M-Bus.
 
-It is not so important if you have used any of those technologies before. What we need is a skilled backend developer
+It is not important if you have used any of those technologies before. What we need is a skilled backend developer
 who is eager to learn new stuff.
 
 Wise Home is based in Ry, near Aarhus, Denmark. The development setup is decentralized and the developers work remote
-around four of five days per week.
+around four or five days per week.
```

Comment:
This is meant as "4 out of the 5 days in the week". Would it be more clear with "...around four out of five days.", and skip the "per week"?

Reply:
Hmm now that I'm thoroughly reading the sentence and not just looking for typos it's a bit perplexing really :) The position is remote but you meet up almost every week? So working remote in Sweden for instance would be a bit of a challenge? I would probably rephrase it then to make it more clear where you expect applicants to sit.

"work remote around four days of the week." sounds fine to me.